protocol/x/subaccounts/keeper/transfer_test.go

```diff
@@ -1407,48 +1388,26 @@ func TestDistributeFees(t *testing.T) {
 			affiliateRevShareAcctAddr:     "",
 			marketMapperRevShareAcctAddr:  constants.AliceAccAddress.String(),
 			unconditionalRevShareAcctAddr: "",
-			revshareParams: revsharetypes.MarketMapperRevenueShareParams{
-				Address:         constants.AliceAccAddress.String(),
-				RevenueSharePpm: 100_000, // 10%
-				ValidDays:       240,
-			},
-			setRevenueShare:    true,
-			revShareExpiration: 100,
-		},
-		"success - market mapper rev share expired": {
```

Comment:
Removing this since its an e2e test. Added an equivalent test in revshare_test.go

Reply:
The test in revshare_test.go broke - looks like there was already an existing test which was commented out